data/projects.js

```diff
@@ -147,5 +147,119 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Web", "Blog"],
 		"markdown": "coala_web.md"
+	},
+	{
+		"name" : "Optimize Caching",
```

Comment:
Are we limiting this to just caching optimization? This could be wider (but makes it more difficult) - improving caching could be *one* of the performance optimizations.